web/src/scripts/webflow/methods.mjs

```diff
@@ -6,7 +6,7 @@ if (typeof process != "undefined") {
 }
 
 // eslint-disable-next-line no-undef
-if (process.env.WEBFLOW_API_TOKEN == undefined) {
+if (process.env.WEBFLOW_API_TOKEN === undefined) {
```

Comment:
[boulder] We should decide whether to use `WEBFLOW_API_TOKEN` or `WEBFLOW_API_TOKEN_V2`.

Reply:
fixed

Reply:
you have have to update the keys in circle ci correct if you use v2?

Reply:
Yes, we have V2 env variables in CircleCi.

Reply:
@seidior I think this is fine. We're just referencing the env variable, and the meaningful change that populates this is in `config.yml`.